regtests/ww3_ufs1.1/input/grdset_a

```diff
@@ -0,0 +1,3 @@
+MODEL: glo_1deg
+INPUT: glo_30m
```

Comment:
These tests are checking ww3_multi with input grids, and I wanted to have it ~= the computational grids, so we can check all the functionality on this scale. I have taken the ice, current and wind from ufs-weather tests to be consistent with them. Of course these are not exactly the same as ufs-weather.
For ww3_gint and ww3_grib, I did the same, just to check if all functions/interpolation are working fine.

Reply:
Because of the other issues, can you confirm that the glo_1deg is fully covered by the glo_30m or should this also be revisited with the needed input change in ufs1.2?